frontend/src/main/components/Courses/CoursesTable.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { useTable, useSortBy } from 'react-table'
+import { Table, Button } from "react-bootstrap";
+import OurTable from "../OurTable";
+
+export default function CoursesTable({ courses }) {
+
+
+    courses = courses.map(course => {
+        course.description = course.description.slice(0, 100) + '...'
```

Comment:
If the description is less than 100 characters, it won't be cut but will still get a '...'